website/docs/r/virtual_machine_extension.html.markdown

```diff
@@ -132,8 +132,8 @@ The following arguments are supported:
 * `name` - (Required) The name of the virtual machine extension peering. Changing
     this forces a new resource to be created.
 
-* `virtual_machine_id` - (Required) The resource ID of the virtual machine. Changing this forces a new
-    resource to be created
+
+* `virtual_machine_id` - (Required) The resource ID of the virtual machine. Changing this forces a new resource to be created
```

Comment:
we avoid the phrase Resource ID since it's ambiguous between Azure and Terraform - can we make this:

```suggestion
* `virtual_machine_id` - (Required) The ID of the Virtual Machine. Changing this forces a new resource to be created
```